packages/bpk-component-card/src/BpkCard.js

```diff
@@ -24,10 +26,18 @@ import STYLES from './BpkCard.scss';
 
 const getClassName = cssModules(STYLES);
 
-const BpkCard = props => {
-  const classNames = [getClassName('bpk-card')];
+type Props = {
+  children: any,
```

Comment:
Had to set this to `any` which is not ideal and needs to be looked at more as setting this to the correct type `Node` as done in other components throws the following error:

```
Cannot create BpkCard element because JSX text [1] is incompatible with Node type [2] in property children.
[incompatible-type]
```

This is the same for other `children` props that are set to any since adding flow

Reply:
Turns out I was not importing `type Node` from React so was using some built in `Node` type so fixed this